app/assets/stylesheets/govuk_publishing_components/components/_search.scss

```diff
@@ -168,10 +168,9 @@ $large-input-size: 50px;
     // Ensure outline appears outside of the element
     outline-offset: 0;
 
-    // no need for black outline as there is enough contrast
-    // with the blue background
     &:focus {
-      box-shadow: none;
+      @extend %gem-c-search-input-focus;
```

Comment:
Have changed this to use a mixin instead which is more inline with how we do focus states elsewhere.

Reply:
![Screenshot 2023-09-26 at 17 08 08](https://github.com/alphagov/govuk_publishing_components/assets/3727504/c3c046da-0393-4a72-a34b-3153012b97b5)
 using the design system focus on the input results in this, I suppose because it's intended for text and not inputs?

Reply:
Thanks @patrickpatrickpatrick! As discussed, the issue seen in your second comment has now been resolved.